src/content/ui-api/gsctl/migrate.md

```diff
@@ -0,0 +1,57 @@
+---
+linkTitle: Migration to kubectl gs
+title: Migrating from gsctl to kubectl gs
+description: gsctl is a CLI for the deprecated Giant Swarm Rest API. As you move from the Rest API towards the Management API, you'll have to say goodby to gsctl and embrace kubectl-gs, step by step. This page should help you make the transition smoothly.
+weight: 1
+menu:
+  main:
+    parent: uiapi-gsctl
+user_questions:
+- What commands in kubectl-gs replace what gsctl commands?
+- How can I replace gsctl with kubectl-gs?
```

Comment:
Should we call it `kubectl gs` or `kubectl-gs`?

Reply:
I settled for kubectl-gs when it comes to the tool's name and `kubectl gs ...` only when it's about invoking a command.